src/ir/compilation_unit/path.rs

```diff
@@ -58,6 +58,17 @@ impl TypePath {
         }
     }
 
+    pub fn to_delimited_string(&self, ns_delimiter: String, name_delimiter: String) -> String {
```

Comment:
`ns_delimiter` and `name_delimiter` should probably be `&str`, not `String`

Reply:
Is there actually any reason why we should even take the delimiters as arguments? Why not just hardcode them?

Reply:
I think I need to move this function to the Js backend, I will move and hardcode